zhaquirks/__init__.py

```diff
@@ -66,26 +66,22 @@ class LocalDataCluster(CustomCluster):
 
     async def bind(self):
         """Prevent bind."""
-        self.debug(f"binding LocalDataCluster with cluster_id: {self.cluster_id}")
+        self.debug("binding LocalDataCluster")
         return (foundation.Status.SUCCESS,)
 
     async def unbind(self):
         """Prevent unbind."""
-        self.debug(f"unbinding LocalDataCluster with cluster_id: {self.cluster_id}")
+        self.debug("unbinding LocalDataCluster")
         return (foundation.Status.SUCCESS,)
 
     async def _configure_reporting(self, *args, **kwargs):  # pylint: disable=W0221
         """Prevent remote configure reporting."""
-        self.debug(
-            f"configuring reporting for LocalDataCluster with cluster_id: {self.cluster_id}",
-            *args,
-            **kwargs,
-        )
+        self.debug("configuring reporting for LocalDataCluster", *args, **kwargs)
```

Comment:
Does this work with `*args` and `**kwargs`?

Reply:
![alt text](image.png)

Reply:
I don't think we can just pass additional arguments and not do anything with them though. How are they expected to appear in a log message?

Did a quick test now and the added log message just seems to break attribute reporting configuration completely:
```python
self = <Logger zigpy.zcl (DEBUG)>, level = 10, msg = '[%s:%s:0x%04x] configuring reporting for LocalDataCluster'
args = ('0x1234', 1, 2820, [AttributeReportingConfig(direction=0, attrid=0x050B, datatype=<DataTypeId.int16: 41>, min_interval=10, max_interval=20, reportable_change=1)]), kwargs = {'manufacturer': None, 'stacklevel': 4}

    def log(self, level, msg, *args, **kwargs):
        """
        Log 'msg % args' with the integer severity 'level'.
    
        To pass exception information, use the keyword argument exc_info with
        a true value, e.g.
    
        logger.log(level, "We have a %s", "mysterious problem", exc_info=True)
        """
        if not isinstance(level, int):
            if raiseExceptions:
                raise TypeError("level must be an integer")
            else:
                return
        if self.isEnabledFor(level):
>           self._log(level, msg, args, **kwargs)
E           TypeError: Logger._log() got an unexpected keyword argument 'manufacturer'

/opt/homebrew/Cellar/python@3.12/3.12.7/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/__init__.py:1609: TypeError
```

Reply:
Thanks for saving me :) I am in the office today so I couldn't test it directly.